parser/src/java/org/apache/hadoop/hive/ql/parse/HiveParser.g

```diff
@@ -1971,10 +1971,12 @@ tableFileFormat
       -> ^(TOK_TABLEFILEFORMAT $inFmt $outFmt $inDriver? $outDriver?)
       | KW_STORED KW_BY storageHandler=StringLiteral
          (KW_WITH KW_SERDEPROPERTIES serdeprops=tableProperties)?
-      -> ^(TOK_STORAGEHANDLER $storageHandler $serdeprops?)
+         (KW_STORED KW_AS fileformat=identifier)?
```

Comment:
Does this mean we will now enable STORED BY and STORED AS in the same query for all storage handlers, like Kudu, Kafka, etc? Do we expect them to break as a result, e.g. if saying something like `stored by kudu stored as orc`? In addition, I guess we should update the Hive language manual too after this change to signify that stored as/stored by are not mutually exclusive.

Reply:
Ah, I just saw below that it shouldn't affect those storage handlers which don't implement new interface method. So the only consideration is then is the language manual update